.github/workflows/release.yml

```diff
@@ -0,0 +1,360 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - .*
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    name: ${{ matrix.platform.name }} ${{ matrix.config.name }}
+    runs-on: ${{ matrix.platform.os }}
+
+    env:
+      CMAKE_CXX_COMPILER_LAUNCHER: ccache # Use ccache to cache C++ compiler output
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+        - { name: Windows VS2022 x86, os: windows-2025 }
+        - { name: Windows VS2022 x64, os: windows-2025 }
+        - { name: Windows MinGW x86,  os: windows-2025, flags: -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ }
+        - { name: Windows MinGW x64,  os: windows-2025, flags: -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ }
+        - { name: Linux GCC,          os: ubuntu-24.04 }
+        - { name: macOS x64,          os: macos-13 }
+        - { name: macOS arm64,        os: macos-15 }
+        config:
+        - { name: Shared, flags: -DBUILD_SHARED_LIBS=ON }
+        - { name: Static, flags: -DBUILD_SHARED_LIBS=OFF }
+        type:
+        - { name: Release }
+        - { name: Debug, flags: -DCMAKE_BUILD_TYPE=Debug }
+        include:
+        - platform: { name: macOS x64, os: macos-13 }
+          config: { name: Frameworks, flags: -GNinja -DSFML_BUILD_FRAMEWORKS=ON -DBUILD_SHARED_LIBS=ON }
+          type: { name: Release }
+        - platform: { name: macOS arm64, os: macos-15 }
+          config: { name: Frameworks, flags: -GNinja -DSFML_BUILD_FRAMEWORKS=ON -DBUILD_SHARED_LIBS=ON }
+          type: { name: Release }
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Set Visual Studio Architecture
+      if: contains(matrix.platform.name, 'Windows VS')
+      uses: ilammy/msvc-dev-cmd@v1
+      with:
+        arch: ${{ contains(matrix.platform.name, 'x86') && 'x86' || 'x64' }}
+
+    # Although the CMake configuration will run with 3.24 on Windows and 3.22
+    # elsewhere, we install 3.25 on Windows in order to support specifying
+    # CMAKE_MSVC_DEBUG_INFORMATION_FORMAT which allows CCache to cache MSVC object
+    # files, see: https://cmake.org/cmake/help/latest/release/3.25.html#variables
+    - name: Get CMake and Ninja
+      uses: lukka/get-cmake@latest
+      with:
+        cmakeVersion: ${{ runner.os == 'Windows' && '3.25' || '3.22' }}
+        ninjaVersion: latest
```

Comment:
This code is import in the CI pipeline because we want to test with the lowest possible CMake version. Doing so ensures that we don't accidentally use features from a newer version. However here in the CD pipeline, we're not longer trying to test the code and rather just want to ship it. I think we can omit all this and use the default CMake version or at least we can simplify this down to just using 3.25 on all machines without the conditional logic.